sample-lnd.conf

```diff
@@ -1306,6 +1306,19 @@
 ; Set to disable blinded route forwarding.
 ; protocol.no-route-blinding=false
 
+; Set to handle messages of a particular type that falls outside of the
+; custom message number range (i.e. 513 is onion messages)
+; protocol.custom-message=
+
+; Specifies feature bits to advertise in the node's init message.
+; protocol.custom-init=
+
+; Specifies custom feature bits to advertise in the node's announcement message.
+; protocol.custom-nodeann=
+
+; Specifies custom feature bits to advertise in the node's invoices.
+; protocol.custom-invoice=
```

Comment:
should we maybe just note that a user can repeat the flags to set multiple bits? Else I think it comes across like they should specify the encoded bit vector

Reply:
That makes sense, just improved those docs.